main.go

```diff
@@ -187,6 +187,11 @@ func (e *expander) expandEnviron(decrypt bool, nofail bool) error {
 		}
 
 		if parameter != nil {
+			// Ensure that this is a valid SSM parameter that we can actually resolve.
+			if !strings.HasPrefix(*parameter, "/") {
+				return fmt.Errorf("SSM parameters must have a leading '/' (ssm:///<path>): %s", envvar)
+			}
```

Comment:
Should this respect `nofail`?

Reply:
I don't know. I thought the intention behind nofail was to bypass aws lookups, not validation. We've already got some other checks like this: https://github.com/remind101/ssm-env/blob/4cbfae786637f867266e5667dd156c7517bb9d99/main.go#L183-L187

Reply:
I guess the question is "what does nofail actually mean?"

Reply:
The arg docs say "Don't fail if error retrieving parameter", which to me implies that this should fail.

Reply:
Subjective of course but I would interpret that to mean this shouldn't fail...